04_environment/infra.tex

```diff
@@ -0,0 +1,212 @@
+\chapter{Environment Infrastructure}
+\section{Foraging}
+\subsection{Inputs, Outputs and Functions}
+
+Foraging has been implemented in the code base. It determines the method of foraging. In short, it is either deer hunting being payoff focused or fishing being a risk averse method of foraging. Deer hunting is linked to a population model, based on a logistic population model. The high level inputs include island IDs, resource contributions from those islands and the chosen foraging method. The high level output is a foraging report which include forage type, input resources, number of participants, number caught, total utility, catch sizes, turn.\newline
```

Comment:
```suggestion
Foraging has been implemented in the code base. It determines the method of foraging. In short, it is either deer hunting being payoff focused or fishing being a risk averse method of foraging. Deer hunting is linked to a logistic population model. When foraging starts, all islands are asked for their resource contributions and chosen foraging method. The high level output is a foraging report which include forage type, input resources, number of participants, number caught, total utility, catch sizes, turn.\newline
```

Did you mean logarithmic instead of logistics?

Reply:
Hey, 
I was informed that it is logistics,